### PR TITLE
Add FAQ

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

**Review by ShifaSZ — CHANGES_REQUESTED** [review]

File: ui/pages/Public/components/Faq.jsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+import { WORKSPACE_REQUIREMENTS } from 'shared/components/panel/LoadWorkspaceDataForm'
+import { SeqrAvailability } from './LandingPage'
+
+export default () => (
+  <Segment basic padded="very">
+    <Header dividing content="FAQ" size="huge" />
+
+    <Header content="Q. What is seqr?" size="medium" />
+    seqr is an open source, Federal Information Security Management Act (FISMA) compliant genomic data analysis platform
```
[the remark this answers is not in the document]
Comment (hanars) [reply ui/pages/Public/components/Faq.jsx:14]: no

File: ui/pages/Public/components/Faq.jsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    *AnVIL is a controlled-access NIH-designated data repository supported on the Terra platform. Users are expected to
+    ensure that data use and sharing within a Terra or AnVIL Workspace are conducted in accordance with all applicable
+    national, tribal, and state laws and regulations, as well as relevant institutional policies and procedures for
+    handling genomic data. However, because seqr runs within Terra or AnVIL workspaces, no additional regulatory
+    approval is required to use seqr to analyze data stored on Terra or AnVIL. <br />
+
+    To learn more about generating a joint called vcf, please refer to
+    this <a href="https://drive.google.com/file/d/1aE7vUvUOZw_r78Osjn1Q0Cs3c5DCuonz/view?usp=sharing" target="blank">documentation</a>
+
+    <Header content="Q. I am unable to log in or access my project in seqr. What should I do?" size="medium" />
```
[the remark this answers is not in the document]
Comment (hanars) [reply ui/pages/Public/components/Faq.jsx:87]: yes this was intentional

File: ui/shared/components/panel/LoadWorkspaceDataForm.jsx [hunk 3]
```diff
@@ -30,6 +30,20 @@ import AnvilFileSelector from 'shared/components/form/AnvilFileSelector'
 
 const VCF_DOCUMENTATION_URL = 'https://storage.googleapis.com/seqr-reference-data/seqr-vcf-info.pdf'
 
+export const WORKSPACE_REQUIREMENTS = [
+  '"Writer" or "Owner" level access to the workspace',
+  'The "Can Share" permission enabled for the workspace',
+  (
+    <span>
+      No &nbsp;
```
[the remark this answers is not in the document]
Comment (hanars) [reply ui/shared/components/panel/LoadWorkspaceDataForm.jsx:38]: No, I used Lynn's formatting as a jumping off point but I don't think it looked good here IRL

File: ui/pages/Public/components/Faq.jsx [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+        navigate to the workspace associated with the project, then select &quot;Data&quot; &gt; &quot;Files&quot; &gt;
+        &quot;Analyze in seqr&quot;.
+      </List.Item>
+    </List>
+
+    <Header content="Q. How long does it take to load data in seqr?" size="medium" />
+    Genomic datasets are large and the seqr loading pipeline richly annotates the variants so data loading can take from
+    a few days to up to a week to process, depending on the sample numbers and data types.
+
+    <Header content="Q. How can I set up seqr locally?" size="medium" />
```
[the remark this answers is not in the document]
Comment (hanars) [reply ui/pages/Public/components/Faq.jsx:112]: Actually I think it makes more sense to move it up, right under the "How can I analyze my data in seqr?" that mentions running locally

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**